docs/skopeo-standalone-sign.1.md

```diff
@@ -4,7 +4,7 @@
 skopeo\-standalone-sign - Debugging tool - Publish and sign an image in one step.
 
 ## SYNOPSIS
-**skopeo standalone-sign** _manifest docker-reference key-fingerprint_ **--output**|**-o** _signature_
+**skopeo standalone-sign** [*options*] _manifest_ _docker-reference_ _key-fingerprint_
```

Comment:
It’s somewhat attractive to change the `--help` output instead (the`--output` option is mandatory), OTOH it would not be worth making the script any more complex.